### DISCUSSION TITLE
Extra line breaks (double-spacing) occur on acceptance of suggestion

### DISCUSSION THREAD

**Discussion by abergerSRS** [Genuitec/Copilot4Eclipse-community#61] [Help]
The ghost text appears properly formatted as single-spaced lines, but upon acceptance of the Copilot suggestion (with Tab), the inserted lines are double-spaced (with empty lines on every other line). 

![image](https://github.com/user-attachments/assets/1bbf2007-7e21-4073-8765-45cf237d6cfb)

![image](https://github.com/user-attachments/assets/f18b93c3-2cc1-4979-9642-8fbd7da51f85)

Is Copilot4Eclipse fighting with the IDE's own auto-completion?

I am using
MCUXpresso IDE v11.10.0 [Build 3148] [2024-07-03]

Eclipse
Version: 2.3.1600.v20231201-0110
Build id: I20231201-0110

Copilot4Eclipse
Version: 1.2.1

**Comment by wayneparrott** [comment]
Thanks for sharing this issue. I have opened a bug report with your info. My understanding is that Copilot4Eclipse attempts to use the EOL settings from the editor as completion content is inserted into the editor (underlying document). We will research this issue and follow up as soon as we know more.

Wayne
Copilot4Eclipse Team

**Comment by wayneparrott** [comment]
@abergerSRS can you try a quick experiment?

1) In the same editor and file, select 2 lines of code. 
2) Select the format action.

Is the resulting format of the 2 lines goofed up with extra lines?

W

**Reply by abergerSRS** [reply]
No. The selected lines are left unchanged by the Format action.

**Reply by wayneparrott** [reply]
Thx for checking.

**Comment by axbusch** [comment]
do you use the https://github.com/wangzw/CppStyle plugin?

i have the same anoying error if i use this plugin (which is handy for clang-format)  but not if i use the copilot plugin  solo.

**Reply by wayneparrott** [reply]
@axbusch thx for sharing this issue with the CppStyle plugin. I'm opening an issue for the team investigate.

**Reply by abergerSRS** [reply]
Yes, I do use CppStyle. Good catch
